### ISSUE TITLE
Cannot build with Carthage 0.25, Xcode 9, Swift 3.2

### ISSUE THREAD

**Issue by mhillebrand206** [0xced/XCDYouTubeKit#345]
```
*** Building scheme "XCDYouTubeKit Framework" in XCDYouTubeKit Demo.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/mhillebrand/work/NewsApp/code/Carthage/Checkouts/XCDYouTubeKit/XCDYouTubeKit\ Demo.xcworkspace -scheme XCDYouTubeKit\ Framework -configuration Release -derivedDataPath /Users/mhillebrand/Library/Caches/org.carthage.CarthageKit/DerivedData/9.0_9M214v/XCDYouTubeKit/2.5.5 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /Users/mhillebrand/work/NewsApp/code/Carthage/Checkouts/XCDYouTubeKit)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/ml/d9hzjcrd3mv_3hj_3qqf__brkdn63y/T/carthage-xcodebuild.oa22is.log
```

...

```
** BUILD FAILED **

The following build commands failed:
	CompileC /Users/mhillebrand/Library/Caches/org.carthage.CarthageKit/DerivedData/9.0_9M214v/XCDYouTubeKit/2.5.5/Build/Intermediates.noindex/XCDYouTubeKit.build/Release-iphoneos/XCDYouTubeKit\ Framework.build/Objects-normal/armv7/XCDYouTubeLogger.o XCDYouTubeKit/XCDYouTubeLogger.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
```

**Comment by thiagoperes** [comment]
+1

**Comment by sixty8** [comment]
+1

**Comment by 0xced** [comment]
I have released version 2.5.6 with Xcode 9 support.
